app/views/admin/schools/_navigation.html.erb

```diff
@@ -1,11 +1,11 @@
-<%= render TabsComponent.new do |component| %>
-  <%= component.tab(path: admin_school_path(@school)) do %>
+<%= render SubnavComponent.new do |component| %>
+  <%= component.nav_item(path: admin_school_path(@school)) do %>
     <strong>Overview</strong>
```

Comment:
```suggestion
    Overview
```

Reply:
I thought I got rid of these!

Reply:
oh wait this isn't part of this PR. i'll change anyway 😆

Reply:
Thanks!